### PR TITLE
Adds Holographic Sex Barriers because I saw a heated debate on Discord.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

image

Adds Sex holographic barriers to warn people that there is sex in the next room. It doesn't block air or movement, but it does warn people, and knowing is half the battle.

## Why It's Good For The Game

image

## Proof Of Testing

See above.

## Changelog

:cl: BurgerBB
balance: Adds Sex holographic barriers to warn people that there is sex in the next room. It doesn't block air or movement, but it does warn people, and knowing is half the battle. They can be purchased in the LustWish under the premium section.
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
this is simultaneously the best and worst pr ive seen all fuckin year. gold star

**Comment by lessthnthree** [comment]
This is hilarious. The barrier could say DND instead of SEX if you wanted to be less obvious, similar to hotel room door signs.

**Comment by BurgerLUA** [comment]
it must say sex

**Comment by lessthnthree** [comment]
> it must say sex

Then it shall be

File: modular_zubbers/code/modules/fluff/sex_barrier.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	name = "holographic sex projector"
+	desc = "Despite the name, it does not project sex, however it does create a non-blocking barrier that informs those who wish to enter that there is sex inside."
+	max_signs = 4
+	creation_time = 10
+	holosign_type = /obj/structure/holosign/sexsign
+	//The janitor sign is also purple so no need to make custom sprites here.
+
+/obj/structure/holosign/sexsign
+	name = "sex sign"
+	desc = "The words flicker DON'T SEX OPEN INSIDE. I think this means that there is sex beyond this door and that you should probably not enter, unless of course you are prepared for unforeseen consequences.
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/fluff/sex_barrier.dm:11]:
```suggestion
	desc = "The words flicker DON'T SEX OPEN INSIDE. I think this means that there is sex beyond this door and that you should probably not enter, unless of course you are prepared for unforeseen consequences.“
```

Comment (StrangeWeirdKitten) [reply modular_zubbers/code/modules/fluff/sex_barrier.dm:11]: Yeah I screwed up my suggestion.

Comment (Majkl-J) [reply modular_zubbers/code/modules/fluff/sex_barrier.dm:11]: You put the wrong airquote symbol there. Fixed it for ya

**Pull request merged**